test/integration/targets/ec2_group/tasks/main.yml

```diff
@@ -325,6 +325,31 @@
           - 'not result.changed'
           - 'result.group_id.startswith("sg-")'
 
+    - name: add a rule that auto creates another security group
```

Comment:
Also, when possible, `resource_prefix` should be included in the names of resources so we can more easily identify which test run they came from, should they survive beyond the test run.